sdk/eventhub/azure-eventhub/TROUBLESHOOTING.md

```diff
@@ -151,17 +211,41 @@ This error is expected when load balancing occurs after EventHubConsumerClient i
 
 However, if no instances are being added or removed, there is an underlying issue that should be addressed.  See [Partition ownership changes a lot](#partition-ownership-changes-a-lot) for additional information and [Filing GitHub issues](#filing-github-issues).
 
+### Blocking Calls in Async
+
+When working with the async client, applications should ensure that they are not blocking the event loop as the load balancing operation runs as a task in the background. Blocking the event loop can impact load balancing, error handling, checkpointing, and recovering after an error. CPU-bound code that can block the event loop should be run in [executor](https://docs.python.org/3/library/asyncio-dev.html#running-blocking-code)
+
 ### High CPU usage
 
 High CPU usage is usually because an instance owns too many partitions.  We recommend no more than three partitions for every 1 CPU core; better to start with 1.5 partitions for each CPU core and test increasing the number of partitions owned.
 
-### Processor client stops receiving
+### One or more partitions have high latency for processing
+
+When processing for one or more partitions is delayed, it is most often because an event processor owns too many partitions. Consider adding additional consumers to handle partitions, we generally recommend 1 - 3 partitions per consumer depending on throughput.
+
+### A partition is not being processed
+
+An event consumer runs continually in a host application for a prolonged period. Sometimes, it may appear that some partitions are uncrowned or are not being processed. Most often, this presents as [Partions close and initialize frequently](#partitions-close-and-initialize-frequently) or there are other errors/warning being raised related to the problem.
```

Comment:
typo 
```suggestion
An event consumer runs continually in a host application for a prolonged period. Sometimes, it may appear that some partitions are uncrowned or are not being processed. Most often, this presents as [Partitions close and initialize frequently](#partitions-close-and-initialize-frequently) or there are other errors/warning being raised related to the problem.
```